mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb

```diff
@@ -380,7 +380,7 @@ func ResourceContainerCluster() *schema.Resource {
 							Computed:     true,
 							AtLeastOneOf: addonsConfigKeys,
 							MaxItems:     1,
-							Description:  `Whether this cluster should enable the Google Compute Engine Persistent Disk Container Storage Interface (CSI) Driver. Defaults to enabled; set disabled = true to disable.`,
+							Description:  `The Compute Engine persistent disk CSI Driver is enabled by default on newly created clusters for the following versions: Linux clusters: GKE version 1.18.10-gke.2100 or later, or 1.19.3-gke.2100 or later.`,
```

Comment:
Whether this cluster should enable the Google Compute Engine Persistent Disk Container Storage Interface (CSI) Driver.

**Note:**: The Compute Engine persistent disk CSI Driver is enabled by default on newly created clusters for the following versions: Linux clusters: GKE version 1.18.10-gke.2100 or later, or 1.19.3-gke.2100 or later.

Reply:
Do you mean leaving both descriptions on this field?

Reply:
Yeah. And adding the note.